tests/components/home_connect/test_diagnostics.py

```diff
@@ -26,10 +29,33 @@ async def test_async_get_config_entry_diagnostics(
     get_appliances: MagicMock,
     snapshot: SnapshotAssertion,
 ) -> None:
-    """Test setup and unload."""
+    """Test config entry diagnostics."""
     get_appliances.side_effect = get_all_appliances
     assert config_entry.state == ConfigEntryState.NOT_LOADED
     assert await integration_setup()
     assert config_entry.state == ConfigEntryState.LOADED
 
     assert await async_get_config_entry_diagnostics(hass, config_entry) == snapshot
+
+
+@pytest.mark.usefixtures("bypass_throttle")
+async def test_device_diagnostics(
```

Comment:
```suggestion
async def test_async_get_device_diagnostics(
```
to stay consistent with the other test naming